### PR TITLE
Bump version to 0.16.1

### PR DESCRIPTION
**Pull request by oviron**
Some updates in versions, libs & plugins to stay up-to-date with Finch

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  scalaVersion := "2.12.2",
-  crossScalaVersions := Seq("2.11.11", "2.12.2")
+  version := "0.16.0",
+  scalaVersion := "2.12.4",
+  crossScalaVersions := Seq("2.11.12", "2.12.4")
 )
 
-lazy val finchVersion = "0.15.0"
+lazy val finchVersion = "0.16.0"
 lazy val finagleOAuth2Version = "0.6.45"
```
Comment (ImLiar) [inline build.sbt]: This version of finagle-oauth2 leads to conflict between finagle 17.12.0 and 6.45.

I think https://github.com/finagle/finagle-oauth2 should be updated first

**Comment by ImLiar** [comment]
/cc @vkostyukov 

How hard would be to update finagle-oauth2 to finagle 17.12.0?

**Comment by ImLiar** [comment]
Ok, it was easy.
This one should be resolved first https://github.com/finagle/finagle-oauth2/pull/27

File: build.sbt (hunk 1 above)
Comment (vkostyukov) [reply build.sbt]: This needs to point to 17.12.0 (which is on its way to MC).

Comment (vkostyukov) [reply build.sbt]: Should be good to go now: https://github.com/finagle/finagle-oauth2/releases/tag/17.12

**Comment by oviron** [comment]
Now it should be OK. Updated finagle-oauth2 to 17.12.0 and by the way bumped finch to 0.16.1
@ImLiar @vkostyukov

**Comment by ImLiar** [comment]
👍

**Comment by vkostyukov** [comment]
Thanks, @oviron! I will do a release today.

**Comment by vkostyukov** [comment]
The release is published: https://github.com/finch/finch-oauth2/releases/tag/0.16.1

**Comment by oviron** [comment]
Great! Thanks @vkostyukov

**Pull request merged**